crates/rpc/rpc/src/otterscan.rs

```diff
@@ -155,10 +158,48 @@ where
         }
 
         // Crop receipts and transform them into OtsTransactionReceipt
-        let timestamp = u64::try_from(block.header.timestamp).unwrap_or(u64::MAX);
+        let timestamp = Some(u64::try_from(block.header.timestamp).unwrap_or(u64::MAX));
         let receipts = receipts
             .drain(page_start..page_end)
-            .map(|receipt| OtsTransactionReceipt { receipt, timestamp })
+            .map(|receipt| {
+                let TransactionReceipt {
```

Comment:
we should really upstream some conversion helpers for this lol

Reply:
https://github.com/alloy-rs/alloy/pull/460

Reply:
beast mode